### PR TITLE
Add Rule,Remediation and Test for SLES-15-030770

### PR DESCRIPTION
**Pull request by yarunachalam**
#### Description:

- Add Rule,Remediation and Test for SLES-15-030770

#### Rationale:

- Add SLE15 rule to 'Record Attempts to Alter Process and Session Initiation Information wtmp'

### CODE REVIEW

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_session_events_wtmp/rule.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    <pre> -w /var/log/wtmp -p wa -k session</pre>
+
+rationale: |-
+    Manual editing of these files may indicate nefarious activity, such
+    as an attacker attempting to remove evidence of an intrusion.
+
+severity: medium
+
+identifiers:
+    cce@sle15: CCE-85717-7
```
Comment (vojtapolasek) [inline linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_session_events_wtmp/rule.yml:28]: You are already using this CCE in a different rule. You can find it by running:
```
./build_product sle15
cd build
ctest -j4 -L quick
```
and inspecting test results.

**Review by vojtapolasek — CHANGES_REQUESTED** [review]
Thank you for this rule. I have just a few minor remarks, see comments.

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_session_events_wtmp/rule.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    srg: SRG-OS-000472-GPOS-00217
+    disa: CCI-000172
+    nist: AU-12(c),AU-12.1(iv)
+
+ocil_clause: 'Audit rule is not present'
+
+ocil: |-
+    To Check the file is being audited by performing the following command
+    <pre> sudo auditctl -l | grep -w '/var/log/wtmp'</pre>
+template:
```
[the remark this answers is not in the document]
Comment (yarunachalam) [reply linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_session_events_wtmp/rule.yml:42]: done

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_session_events_wtmp/rule.yml (hunk 1 above)
Comment (yarunachalam) [reply linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_session_events_wtmp/rule.yml]: Fixed

**Review by vojtapolasek — APPROVED** [review]

**Pull request merged**